### PR TITLE
Updates to database schema

### PR DESCRIPTION
**Pull request by dave-mills**
While making this db schema diagram for the site, I realised there were a bunch of missing foreign-key links from the database. This PR fixes all that!

It does 2 main things
 - updates all INT ids to use BIGINT (as since Laravel 5.8, this has been the default. Until now we've had some id fields use INT and others use BIGINT, which makes some foreign key links impossible...)
 - adds in all the required foreign keys (see diagram)

https://www.lucidchart.com/invitations/accept/91cf7d43-1b71-4c6c-bdff-3e76a0000114

### CODE REVIEW

File: database/migrations/2020_04_08_102803_make_all_ids_big_ints_because_laravel_changed_defaults_and_now_all_future_autogenerated_migrations_with_links_will_fail.php [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+        Schema::table('contactus', function(Blueprint $table) {
+            $table->bigIncrements('id')->change();
+        });
+
+        Schema::table('evaluationquestions', function(Blueprint $table) {
+            $table->bigIncrements('id')->change();
+        });
+
+        Schema::table('events', function(Blueprint $table) {
+            $table->bigIncrements('id')->change();
```
Comment (luciafalcinelli) [inline database/migrations/2020_04_08_102803_make_all_ids_big_ints_because_laravel_changed_defaults_and_now_all_future_autogenerated_migrations_with_links_will_fail.php:170]: In events table, the id type is already BIGINT

File: database/migrations/2020_04_08_102803_make_all_ids_big_ints_because_laravel_changed_defaults_and_now_all_future_autogenerated_migrations_with_links_will_fail.php [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+        });
+
+        Schema::table('failed_jobs', function(Blueprint $table) {
+            $table->bigIncrements('id')->change();
```
Comment (luciafalcinelli) [inline database/migrations/2020_04_08_102803_make_all_ids_big_ints_because_laravel_changed_defaults_and_now_all_future_autogenerated_migrations_with_links_will_fail.php:174]: in failed_jobs table, the id is already BIGINT

File: database/migrations/2020_04_08_102803_make_all_ids_big_ints_because_laravel_changed_defaults_and_now_all_future_autogenerated_migrations_with_links_will_fail.php [hunk 3]
```diff
@@ -0,0 +1,335 @@
...
+        });
+
+        Schema::table('guest_tags', function(Blueprint $table) {
+            $table->bigIncrements('id')->change();
+            $table->bigInteger('user_id')->unsigned()->change();
+            $table->bigInteger('tag_id')->unsigned()->change();
+        });
+
+        Schema::table('jobs', function(Blueprint $table) {
+            $table->bigIncrements('id')->change();
```
Comment (luciafalcinelli) [inline database/migrations/2020_04_08_102803_make_all_ids_big_ints_because_laravel_changed_defaults_and_now_all_future_autogenerated_migrations_with_links_will_fail.php:184]: the jobs is already BIGINT

File: database/migrations/2020_04_08_102803_make_all_ids_big_ints_because_laravel_changed_defaults_and_now_all_future_autogenerated_migrations_with_links_will_fail.php [hunk 4]
```diff
@@ -0,0 +1,335 @@
...
+
+        Schema::table('_link_article_users', function(Blueprint $table) {
+            $table->bigInteger('article_id')->unsigned()->change();
+            $table->bigInteger('user_id')->unsigned()->change();
+        });
+
+        Schema::table('_link_authors_troves', function(Blueprint $table) {
+            $table->bigInteger('author_id')->unsigned()->change();
+            $table->bigInteger('trove_id')->unsigned()->change();
+        });
```
Comment (luciafalcinelli) [inline database/migrations/2020_04_08_102803_make_all_ids_big_ints_because_laravel_changed_defaults_and_now_all_future_autogenerated_migrations_with_links_will_fail.php:87]: In _link_authors_troves the id type is a INT, maybe you should add the following row:  
$table->bigIncrements('id')->change();

File: database/migrations/2020_04_08_102803_make_all_ids_big_ints_because_laravel_changed_defaults_and_now_all_future_autogenerated_migrations_with_links_will_fail.php [hunk 5]
```diff
@@ -0,0 +1,335 @@
...
+
+        Schema::table('_link_collections_troves', function(Blueprint $table) {
+            $table->bigInteger('collection_id')->unsigned()->change();
```
Comment (luciafalcinelli) [inline database/migrations/2020_04_08_102803_make_all_ids_big_ints_because_laravel_changed_defaults_and_now_all_future_autogenerated_migrations_with_links_will_fail.php:90]: In _link_collections_troves the id type is a INT, maybe you should add the following row:
$table->bigIncrements('id')->change();

File: database/migrations/2020_04_08_102803_make_all_ids_big_ints_because_laravel_changed_defaults_and_now_all_future_autogenerated_migrations_with_links_will_fail.php (hunk 1 above)
Comment (dave-mills) [reply database/migrations/2020_04_08_102803_make_all_ids_big_ints_because_laravel_changed_defaults_and_now_all_future_autogenerated_migrations_with_links_will_fail.php]: Good catch - I'd just done a blanket copy-paste update for speed. I've removed these un-needed updates in the latest commit

File: database/migrations/2020_04_08_102803_make_all_ids_big_ints_because_laravel_changed_defaults_and_now_all_future_autogenerated_migrations_with_links_will_fail.php (hunk 4 above)
Comment (dave-mills) [reply database/migrations/2020_04_08_102803_make_all_ids_big_ints_because_laravel_changed_defaults_and_now_all_future_autogenerated_migrations_with_links_will_fail.php:88]: Done - thanks,

**Pull request merged**